Facebook.iOS/ShareKit/source/Facebook.ShareKit/ApiDefinition.cs

```diff
@@ -0,0 +1,1310 @@
+﻿using System;
+
+using UIKit;
+using Foundation;
+using ObjCRuntime;
+using CoreGraphics;
+using Photos;
+
+namespace Facebook.ShareKit {
+	// @interface FBSDKAppGroupAddDialog : NSObject
+	[BaseType (typeof (NSObject),
+		Name = "FBSDKAppGroupAddDialog",
+		Delegates = new [] { "Delegate" },
+		Events = new [] { typeof (AppGroupAddDialogDelegate) })]
+	[Obsolete ("App and game groups are being deprecated. See https://developers.facebook.com/docs/games/services/game-groups for more information.")]
+	interface AppGroupAddDialog {
+
+		// +(instancetype)showWithContent:(FBSDKAppGroupContent *)content delegate:(id<FBSDKAppGroupAddDialogDelegate>)delegate;
+		[Static]
+		[Export ("showWithContent:delegate:")]
+		[Obsolete ("App and game groups are being deprecated. See https://developers.facebook.com/docs/games/services/game-groups for more information.")]
+		AppGroupAddDialog Show ([NullAllowed] AppGroupContent content, [NullAllowed] IAppGroupAddDialogDelegate aDelegate);
+
+		// @property (nonatomic, weak) id<FBSDKAppGroupAddDialogDelegate> delegate;
+		[NullAllowed]
+		[Export ("delegate", ArgumentSemantic.Weak)]
+		[Obsolete ("App and game groups are being deprecated. See https://developers.facebook.com/docs/games/services/game-groups for more information.")]
+		IAppGroupAddDialogDelegate Delegate { get; set; }
+
+		// @property (copy, nonatomic) FBSDKAppGroupContent * content;
+		[NullAllowed]
+		[Export ("content", ArgumentSemantic.Copy)]
+		[Obsolete ("App and game groups are being deprecated. See https://developers.facebook.com/docs/games/services/game-groups for more information.")]
+		AppGroupContent Content { get; set; }
+
+		// -(BOOL)canShow;
+		[Export ("canShow")]
+		[Obsolete ("App and game groups are being deprecated. See https://developers.facebook.com/docs/games/services/game-groups for more information.")]
+		bool CanShow { get; }
+
+		// -(BOOL)show;
+		[Export ("show")]
+		[Obsolete ("App and game groups are being deprecated. See https://developers.facebook.com/docs/games/services/game-groups for more information.")]
+		bool Show ();
+
+		// -(BOOL)validateWithError:(NSError **)errorRef;
+		[Export ("validateWithError:")]
+		[Obsolete ("App and game groups are being deprecated. See https://developers.facebook.com/docs/games/services/game-groups for more information.")]
+		bool Validate (out NSError error);
```

Comment:
I guess this is obsoleted, but maybe then we should just leave it as `ValidateWithError` instead of changing it....